tests/ga/test_extension.py

```diff
@@ -346,24 +346,10 @@ def test_migration_ignores_tree_remove_errors(self, shutil_mock):  # pylint: dis
         return
 
 
-class ExtensionTestCase(AgentTestCase):
```

Comment:
These tests were calling CGroupConfigurator.disable(). There is no need, since the configurator is disabled by default.